chemprop/cli/predict.py

```diff
@@ -39,12 +39,17 @@ def func(cls, args: Namespace):
 
 def add_predict_args(parser: ArgumentParser) -> ArgumentParser:
     parser.add_argument(
-        "-i", "--test-path", required=True, help="Path to an input CSV file containing SMILES."
+        "-i",
+        "--test-path",
+        required=True,
+        type=Path,
+        help="Path to an input CSV file containing SMILES.",
     )
     parser.add_argument(
         "-o",
         "--output",
         "--preds-path",
+        type=Path,
         help="Path to CSV or PICKLE file where predictions will be saved. If the file extension is .pkl, will be saved as a PICKLE file. If not provided and the test_path is /path/to/test/test.csv, predictions will be saved to /path/to/test/test_preds.csv.",
```

Comment:
```suggestion
        help="Path to which predictions will be saved. If the file extension is .pkl, will be saved as a pickle file. Otherwise, will save predictions as a CSV. By default, predictions will be saved to the same location as '--test-path' with '_preds' appended, i.e., 'PATH/TO/TEST_PATH_preds.csv'.",
```

Reply:
Happy to update this help string. Depending on what you think of #599, I could make this change in that PR instead to avoid having to resolve a merge conflict.

Reply:
I've added this to #599